src/common/utils/recommend.ts

```diff
@@ -92,9 +92,13 @@ export function getFedLevel(projection: Projection): FedLevel | null {
 
   if (weeklyCasesPer100k > 100 && positiveTestRate > 0.1) {
     return FedLevel.RED;
+  } else if (weeklyCasesPer100k > 100 || positiveTestRate > 0.1) {
+    return FedLevel.YELLOW;
   } else if (
-    (10 < weeklyCasesPer100k && weeklyCasesPer100k < 100) ||
-    (0.05 <= positiveTestRate && positiveTestRate <= 0.1)
+    10 < weeklyCasesPer100k &&
+    weeklyCasesPer100k < 100 &&
+    0.05 <= positiveTestRate &&
+    positiveTestRate <= 0.1
```

Comment:
In order to get to this `else if`, I think `weeklyCasesPer100k <= 100` and `positiveTestRate <= 0.1` both have to be true already (else you would have hit the previous `else if` block).  So if we want this can be simplified to:

`else if (10 < weeklyCasesPer100k && 0.05 <= positiveTestRate) {`